.ci/test_r_package.sh

```diff
@@ -94,7 +94,12 @@ fi
 
 # Manually install Depends and Imports libraries + 'testthat'
 # to avoid a CI-time dependency on devtools (for devtools::install_deps())
-packages="c('data.table', 'jsonlite', 'Matrix', 'R6', 'testthat')"
+# NOTE: testthat is not required when running rchk
```

Comment:
`{testthat}` has a lot of dependencies, and since the `rchk` job runs on Linux they have to be compiled from source (since CRAN does not provide Linux binaries).

This additional `if` statement adds a bit of new complexity to this CI script, but I think it's worth it because I found in testing on my fork that this saves 1-2 minutes on the run time of the job.